clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java

```diff
@@ -71,7 +72,11 @@ public KubernetesCredentials(KubernetesApiAdaptor apiAdaptor,
 
         HashMap<String, String> secretData = new HashMap<>(1);
         String dockerCfg = String.format("{ \"%s\": { \"auth\": \"%s\", \"email\": \"%s\" } }", account.getV2Endpoint(), account.getBasicAuth(), account.getEmail());
-        dockerCfg = new String(Base64.getEncoder().encode(dockerCfg.getBytes()));
+        try{
```

Comment:
Please put single-space separators between `try` and its opening brace, before and after `catch`, and just before `catch`'s opening brace.